tweetprocessors.js

```diff
@@ -113,13 +115,100 @@ class HashtagStats {
 
   /**
    * Returns a list of all the trending hashtags and countries.
+   * The list is ordered by country, hashtags with highest count.
+   * It is formattd as a list of [country, hashtag, entry].
    */
   get() {
-    // TODO: implement this
+    this.HashtagStats.findAll({order: ['country', 'count']}).then(full_list => {
+      var country_list = [];
+      for (var i = 0; i < full_list.length; i++) {
+        var country = full_list[i].dataValues.country;
+        var hashtag = full_list[i].dataValues.hashtag;
+        var count = full_list[i].dataValues.count;
+        var entry = Object.freeze([country, hashtag, count]);
+        country_list.push(entry);
+      }
```

Comment:
It seems like this loop could be rewritten as a `map` operation, which would look a bit nicer.